packages/php-wasm/universal/src/lib/php-request-handler.ts

```diff
@@ -284,7 +296,10 @@ export class PHPRequestHandler implements RequestHandler {
 		if (this.php.fileExists(resolvedFsPath)) {
 			return resolvedFsPath;
 		}
-		return `${this.#DOCROOT}/index.php`;
+		if (this.php.fileExists(`${this.#DOCROOT}/index.php`)) {
+			return `${this.#DOCROOT}/index.php`;
+		}
+		throw new Error(`File not found: ${resolvedFsPath}`);
```

Comment:
Code style nitpick – let's use [guards](https://en.wikipedia.org/wiki/Guard_(computer_science)) instead of throwing in the default case:

```suggestion
		if (!this.php.fileExists(`${this.#DOCROOT}/index.php`)) {
			throw new Error(`File not found: ${resolvedFsPath}`);
		}
		return `${this.#DOCROOT}/index.php`;
```